web/src/app/escalation-policies/PolicyServicesCard.tsx

```diff
@@ -1,24 +1,31 @@
 import React from 'react'
-import { PropTypes as p } from 'prop-types'
 import Card from '@material-ui/core/Card'
 import { makeStyles } from '@material-ui/core/styles'
-import FlatList from '../lists/FlatList'
+import FlatList, { FlatListListItem } from '../lists/FlatList'
+import _ from 'lodash'
 
 const useStyles = makeStyles(() => ({
   card: {
     width: '100%',
   },
 }))
 
-function PolicyServicesCard(props) {
+interface PolicyServicesCardProps {
+  services: { id: string; name: string }[]
+}
+
+function PolicyServicesCard(props: PolicyServicesCardProps): JSX.Element {
   const classes = useStyles()
 
-  function getServicesItems() {
-    return props.services.map((service) => ({
+  function getServicesItems(): FlatListListItem[] {
+    const items = props.services.map((service) => ({
       title: service.name,
       url: `/services/${service.id}`,
     }))
+
+    return _.sortBy(items, 'title')
```

Comment:
I'm noticing titles starting with 'a' are prioritized after all uppercase items, rather than as `AaBbCc`. I don't see this as a major issue really but we _do_ sort our paginated lists as `AaBbCc`. Does lodash accept any sort options?

Reply:
e.g.

![Screen Shot 2021-04-27 at 2 08 27 PM](https://user-images.githubusercontent.com/11381794/116313035-153ec180-a762-11eb-974c-09f48208558e.png)

Reply:
Something like this might work:
```
_.sortBy(items, [i=>i.title.toLowerCase(), i=>i.title])
```

Reply:
Though our unique constrains are all case-insensitive so `_.sortBy(items, i=>i.title.toLowerCase())` would also be sufficient